src/Compilers/CSharp/Portable/Symbols/Metadata/PE/PENamespaceSymbol.cs

```diff
@@ -190,6 +191,21 @@ protected void LoadAllMembers(IEnumerable<IGrouping<string, TypeDefinitionHandle
             LazyInitializeTypes(nestedTypes);
         }
 
+        private int GetQualifiedNameLength()
```

Comment:
I did not see similar usage outside of PE, while PE symbols do this very frequently, so decided to keep the functionality where it is used.